src/mumble/NetworkConfig.ui

```diff
@@ -292,6 +292,20 @@ Prevents the client from downloading images embedded into chat messages with the
         </property>
        </widget>
       </item>
+      <item>
+       <widget class="QCheckBox" name="qcbHideOS">
+        <property name="toolTip">
+         <string>Prevent OS information being sent to the server</string>
+        </property>
+        <property name="whatsThis">
+         <string>&lt;b&gt;Don't send OS information to the server&lt;/b&gt;&lt;br/&gt;
+Prevents the client from sending potentially identifying information about the operating system to the server.</string>
+        </property>
+        <property name="text">
+         <string>Hide OS information</string>
```

Comment:
This is hard to name precisely and concisely.
How about "Do not send OS information to Mumble servers"?
The name "Hide OS information" seems a little too vague for my tates... It can mean many things.